library/src/blas3/geam_device.h

```diff
@@ -42,7 +42,7 @@ geam_device(
         }
 
         C[c_index] = alpha * A[a_index];
-        C[c_index] += beta * B[b_index];
+        C[c_index] = fma(beta, B[b_index], C[c_index]);
```

Comment:
Thanks.  But again, why prefer two statments to a single fma(beta, B[i], alpha*A[i])

Reply:
Changed to use a single statement with fma.